frontend/src/component/project/Project/ProjectSettings/ProjectActions/ProjectActionsTable/ProjectActionsModal/ProjectActionsForm/ProjectActionsFormStep/ProjectActionsFormStepActions/ProjectActionsFormStepActions.tsx

```diff
@@ -44,6 +46,14 @@ export const ProjectActionsFormStepActions = ({
     validated,
 }: IProjectActionsFormStepActionsProps) => {
     const projectId = useRequiredPathParam('projectId');
+    const { project } = useProjectOverview(projectId);
+    const { features } = useFeatureSearch({ project: `IS:${projectId}` });
```

Comment:
I'm afraid this could be heavy in the db where we just want to get the feature name, but I think that can be a future optimization (and we also don't expect this UI to be used intensively)

Reply:
Agreed. This PR is still an improvement over what we had before, since this is now fetched at a higher level instead of being fetched for each action item.